app/models/document_type.rb

```diff
@@ -0,0 +1,44 @@
+# == Schema Information
+#
+# Table name: tags
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  type       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class DocumentType < Tag
+  DOCUMENT_TYPES = [
```

Comment:
Hmm, not sure if we want to restrict that list

Reply:
Also, that list is funny now, I wouldn't pay too much attention to what is in the data currently. See there is document type `All`, they should clean this up, what they wanted to just tag Law with different types. Maybe we will have to extract some of those tags to be something else in the future (for example if application logic will depend on it, I suspect "Out of date" would be the first candidate)

Reply:
yeah I agree, this list will definitely change. For now I can remove the validation, but eventually, I think we will need some constraint on "document type". I guess this is a more precise term than, let's say "category".

Reply:
Not sure, I think right now those are like categories, and definitely, that list needs cleanup from their side. I would just remove validation for now.